code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.cpp

```diff
@@ -0,0 +1,51 @@
+// Part of Cosmos by OpenGenus Foundation
+
+//dynamic programming || Longest repeating subsequence
+
+#include <iostream>
+#include <cstring>
+
+using namespace std;
+
+const int MAX = 1010;
+
+int dp[MAX][MAX];
+
+int longestRepeatingSubsequence(string s)
+{
+    int n = s.size();  // Obtaining the length of the string
+
+    //Initializing the table
+    for (int x = 0; x <= n; x++)
+    {
+        for (int y = 0; y <= n; y++)
+        {
+            dp[x][y] = 0;
+        }
+    }
+
+    for (int x = 1; x <= n; x++)
+    {
+        for (int y = 1; y <= n; y++)
+        {
+            if (s[x-1] == s[y-1] && x != y)
+            {
+                dp[x][y] = 1 + dp[x-1][y-1];
+            }
+            else
+            {
+                dp[x][y] = max(dp[x-1][y], dp[x][y-1]);
+            }
+        }
+    }
+
+    // Returning the value of the result
+    return dp[n][n];
```

Comment:
I don't know how you will replace this, but do _NOT_ use variable length arrays, as they are not part of standard C++. Find an alternative, such as nested std::vectors.